features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/helpers/edit-asset.js

```diff
@@ -22,17 +22,20 @@ var processData = function(data){
 
 var selectCategory = function(data) {
 
-	var selected, 
+	var selected,
```

Comment:
Avoid using selectCatogory() function and use your own function to manipulate ownerlist